packages/api/package.json

```diff
@@ -36,6 +36,9 @@
     "bullmq": "^1.80.6",
     "ioredis-mock": "^7.2.0"
   },
+  "peerDependencies": {
+    "@bull-board/ui": "5.1.1"
```

Comment:
lets not restrict the version here, make it `*`

Reply:
Wasn't sure about compatibility issues so I copied the version from the inverse dependency (ui's dependany on api@5.1.1)

Changed to *